app/governance/[id]/page.tsx

```diff
@@ -0,0 +1,137 @@
+
+  if (!proposal) {
+    return <div>No proposal found with the ID {params.id}</div>;
+  }
+  console.log(proposal);
+  const isActive = formatProposalStatus(proposal.status)=='ACTIVE';
+
+  const votesData = calculateVotePercentages(proposal.final_tally_result);
+
+  return (
+  <div className={styles.proposalContainer}>
+    <div className={styles.proposalHeaderContainer}>
+        <div className={styles.proposalCard1}>
+          <div style={{borderRight: "1px solid", padding: "10px"}}><Text>#{proposal.proposal_id}</Text></div>
+          <div style={{padding: "10px"}}><Text>{formatProposalStatus(proposal.status)}</Text></div>
+        </div>
+        <div>
+            <Text font='proto_mono' size="x-lg">{proposal.content.title}</Text>
+        </div>
+        <div>
+          <Text opacity={0.4}>{proposal.content.description}</Text>
+        </div>
+    </div>
+    
+    <div className={styles.proposalCardContainer}>
+      <div className={styles.proposalCardContainer1}>
+        <div className={styles.proposalInfoBox}>
+          <div className={styles.proposalInfo}>
+            <div><Text font="proto_mono" opacity={0.3}>Type:</Text></div>
+            <div><Text font="proto_mono">{formatProposalType(proposal.content["@type"])}</Text></div>
+          </div>
+          <div className={styles.proposalInfo}>
+            <div><Text font="proto_mono" opacity={0.3}>Total Deposit:</Text></div>
+            <div>
+              <Text font="proto_mono">{formatDeposit(proposal.total_deposit[0].amount)} </Text>
+            </div>
+          </div>
+          <div className={styles.proposalInfo}>
+              <div><Text font="proto_mono" opacity={0.3}>Turnout/ Quorum: </Text></div>
+              <div><Text font="proto_mono">38.1% 33.4%</Text></div>
+            </div>
+        </div>
+        <div className={styles.proposalInfoBox}>
+            <div className={styles.proposalInfo}>
+              <div><Text font="proto_mono" opacity={0.3}>Submit Time:</Text></div>
+              <div><Text font="proto_mono">{formatTime(proposal.submit_time)}</Text></div>
+            </div>
+            <div className={styles.proposalInfo}>
+              <div><Text font="proto_mono" opacity={0.3}>Voting End Time:</Text></div>
+              <div><Text font="proto_mono">{formatTime(proposal.voting_end_time)}</Text></div>
+            </div>
+        </div>
+        
+      </div>
+      <div className={styles.proposalCardContainer2}>
+        <div className={styles.proposalInfoBoxVoting}>
+          <div className={styles.proposalInfoRow1}>
+            <div className={styles.proposalInfoVoting}>
+              <div className={styles.votingInfoRow1}>
+                
+                <div style={{display: "flex", flexDirection: "row", justifyContent:"space-around"}}> <div className={styles.circle} style={{ backgroundColor: "green", margin:"10px 5px 0px 10px" }}></div> <div><Text font="proto_mono">Yes</Text></div></div>
```

Comment:
this styling is static, so It would be preferred to stylesheets than inline. inline should only be used for dynamic content